internal/controller/kbsconfig_controller.go

```diff
@@ -358,6 +358,19 @@ func (r *KbsConfigReconciler) newKbsDeployment(ctx context.Context) (*appsv1.Dep
 	volumes = append(volumes, *volume)
 	kbsVM = append(kbsVM, volumeMount)
 
+	// attestation policy
+	// ibmse need set attestation policy
```

Comment:
Didn't understand this code comment ?
"ibmse need set attestation policy" -> is this relevant ?

Reply:
removed this comment